applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -876,6 +879,25 @@ void SmallStrainUPwDiffOrderElement::AssignPressureToIntermediateNodes()
             ThreadSafeNodeWrite(rGeom[9],WATER_PRESSURE, 0.5 * (p2 + p3) );
             break;
         }
+        case 15: //2D T15P3
```

Comment:
OK, You are always using a linear interpolation for water flow which is good for partially saturated flow.

Reply:
Right, See the comment above